pkg/R/sparkR.R

```diff
@@ -36,14 +36,17 @@ sparkR.stop <- function(env) {
   if (exists(".sparkRjsc", envir = env)) {
     sc <- get(".sparkRjsc", envir = env)
     callJMethod(sc, "stop")
+    rm(".sparkRjsc", envir = env)
   }
 
```

Comment:
@shivaram Thank you for your reply, this does makes sense to me now. I added the  <tryCatch> to <connectBackend>, and added an error and warning message. 

I also changed the boolean statement in <connExists> to what it was originally.

Then I realized that <connectBackend> does a test if ".sparkRcon" exists, if it does it exists it returns ".sparkRcon" else it connects. I have modified <connectBackend> such that if ".sparkRcon" does exist but is not open then try to connect. If no connection exists throws a warning.